static/js/portico/signup.js

```diff
@@ -196,4 +196,40 @@ $(function () {
     $("body").on("click", "#choose_email .choose-email-box", function () {
         this.parentNode.submit();
     });
+
+    const offer_details_behavior = function () {
+        const selected_object = $("#organization_type_select")[0];
+        const selected_value = selected_object.options[selected_object.selectedIndex].value;
+        const $offer_details = $("#offer_details");
+        const $open_source_offer = $("#open_source_offer");
```

Comment:
We should be consistent in whether these Jquery variable have $ prefix or not. I don't think the prefixing is something we generally do, so probably should be removed